eng/scripts/Language-Settings.ps1

```diff
@@ -101,13 +101,11 @@ function Get-AllPackageInfoFromRepo($serviceDirectory)
   $searchPath = Join-Path $RepoRoot "sdk"
   $pkgFiles = @()
   if ($serviceDirectory) {
-    $searchPath = Join-Path $searchPath $serviceDirectory "go.mod"
-    [array]$pkgFiles = @(Get-ChildItem $searchPath)
-  } else {
-    # If service directory is not passed in, find all modules
-    [array]$pkgFiles = Get-ChildItem -Path $searchPath -Include "go.mod" -Recurse
```

Comment:
Why is this changed? This method is supposed to return all packages in the repo when service directory is empty. right? This is not used by APIView. Are we sure it is not used by any other EngSys scripts?